drivers/usb_c/tcpc/ps8xxx.c

```diff
@@ -581,6 +598,7 @@ void ps8xxx_alert_work_cb(struct k_work *work)
 		return;
 	}
 
+	data->cc_changed = true;
```

Comment:
Why is this needed?  This flags a CC change on any interrupt.  If that's desired, I don't think you need to set `cc_changed=true` below in the while loop.

Reply:
I forgot to remove this after inserting the same in specific cases. Fixed.